runtime-builder/tests/README.md

```diff
@@ -9,15 +9,15 @@ for information on how to write tests.
 
 ## Integration Test
 The \/integration directory contains a test web application for performing an end to end test.
-Refer to integration test framework [README](https://github.com/GoogleCloudPlatform/runtimes-common/tree/master/integration_tests) for the design and requirements. The directory serves as a GOPATH. It also includes all the dependencies for it to be used by the test container. A go-test.yaml is provided as the cloudbuild.yaml for deploying the app and it points to the staging go1-builder image. Every newly built image will have the "staging" tag.
+Refer to integration test framework [README](https://github.com/GoogleCloudPlatform/runtimes-common/tree/master/integration_tests) for the design and requirements. The directory serves as a GOPATH. It also includes all the dependencies for it to be used by the test container. A test.yaml.in is provided as the templated cloudbuild.yaml for deploying the app: the ${STAGING_IMAGE} environment variable will contain the path to the go1-builder staging image to use (this variable must be set). Every newly built image will have the "staging" tag.
 
 To perform the test:
 * Manually run a container build of the go1-builder image. Skip this step if you already have an existing image in GCR that you want to test.
 * Set `gcloud config set app/use_runtime_builders true`
 * Set `gcloud config set app/runtime_builders_root` to the integration/ directory.
-* Run test.sh \<project_id>. This will use the configured go-test.yaml to build and deploy the test app. It will also invoke the test driver to perform the test suite to verify that it is working.
+* Run test.sh \<project_id>. This will use the configured test.yaml to build and deploy the test app. It will also invoke the test driver to perform the test suite to verify that it is working.
```

Comment:
Nit.

s/configured test.yaml/generated test.yaml/

Also, please add another statement at the end ...

Optionally pass in the image tag name as the second parameter to use a different builder image.